### ISSUE TITLE
Typing todo doesn't list commands

### ISSUE THREAD

**Issue by nblackburn** [vesln/todo#17]
Typing todo doesn't list the available commands as stated in [readme.md](https://github.com/vesln/todo/blob/master/README.md)

```
todo                     Print help
todo add Go shopping     Create a new todo item
todo ls                  Print all pending todo items
todo ls @tag             Print todo items containing "@tag"
todo ls ~@tag            Print todo items not containing "@tag"
todo ls --all            Print completed and pending todo items
todo ls --done           Print completed todo items
todo check 1             Mark #1 as completed
todo mv 1 42             Change the id of given todo
todo undo 1              Revert #1 to pending
todo rm 1                Remove #1 item
todo clear               Destroy all todo items
```

**Comment by samuelcouch** [comment]
+1 `todo help` echos the commands.

**Comment by nblackburn** [comment]
@samuelcouch Yup but think this should be the default behavior when typing `todo` as it follows the convention of other packages and cli tools.

**Comment by samuelcouch** [comment]
See #19

**Comment by vesln** [comment]
Resolved, see  #18 

Thanks!
